model/vm_host.rb

```diff
@@ -67,6 +67,30 @@ def ip6_random_vm_network(prefix = 79)
     end
   end
 
+  def ip4_random_vm_network
+    subnet = NetAddr::CIDR.create(ipv4_bucket)
+    ip_list = subnet.enumerate
+  
+    # remove the first and the last IP from the range
```

Comment:
Purpose of this comment is obvious from code. Not obvious: why you want to remove the ends of the range